kodilla-patterns/src/main/java/com/kodilla/patterns/builder/bigmac/Bigmac.java

```diff
@@ -0,0 +1,100 @@
+package com.kodilla.patterns.builder.bigmac;
+
+import java.util.*;
+
+import static com.kodilla.patterns.builder.bigmac.Validator.*;
+
+public final class Bigmac {
+    private final String roll;
+    private final String sauce;
+    private final int burgers;
+    private final Set<String> ingredients;
+
+    private Bigmac(String roll, String sauce, int burgers, Set<String> ingredients) {
```

Comment:
I think it could be a little bit simplified, when we have one private no-arguments constructor: 
```
private Bigmac()
```
and Builder has reference to bigmac:
```
Bigmac out = new Bigmac()
```
thanks to that we don't have to copy all fields to Builder (roll, sauce,...)